_dev/js/back/src/components/panel/payment-mode.vue

```diff
@@ -114,18 +160,27 @@
         this.$store.dispatch('updatePaymentMode', mode).then(() => {
           this.$store.dispatch('getOnboardingLink');
         });
+      },
+      cancelCaptureMode() {
+        var selector = "#intent-mode input[value='" + this.captureMode + "']";
+        document.querySelector(selector).click();
+      },
+      updateCaptureMode() {
+        var captureMode =
+          this.captureMode === 'CAPTURE'
+            ? 'AUTHORIZE'
+            : this.captureMode === 'AUTHORIZE'
+            ? 'CAPTURE'
+            : '';
+        if (captureMode !== '') {
+          this.$store.dispatch('updateCaptureMode', captureMode);
```

Comment:
There is a good practice arround Actions and Mutations:

Define a map of posible actions, mutations
```
export const mutations = {
   SET_FOO = 'SET_FOO',
   SET_BAR = 'SET_BAR',
   SET_BAZ = 'SET_BAZ',
}
```
then on your store:
```
import {mutations} from '<my_fancy_mutations_location>';

const store = {
  mutations: {
    // Even you can export just the symbols to do [SET_FOO] if you dont like the "mutations." part
    [mutations.SET_FOO]: functions () {}
  }
}
```
and finally

```
this.$store.commit(mutations.SET_FOO, payload); // or just SET_FOO as stated before
```

this practice avoid messing up with typos on commits / actions names